AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_PeriodicTests.py

```diff
@@ -0,0 +1,409 @@
+
+        # 8. Close & open previously saved original_material_document, new_material_document, & child_material_document.
+        closed_all_material_documents = atom_tools_utils.close_all_documents()
+        Report.result(
+            Tests.closed_original_new_and_child_material_documents,
+            closed_all_material_documents is True)
+        original_material_document_id = atom_tools_utils.open_document(original_material_document)
+        new_material_document_id = atom_tools_utils.open_document(new_material_document)
+        child_material_document_id = atom_tools_utils.open_document(child_material_document)
+        material_document_ids = [original_material_document_id, new_material_document_id, child_material_document_id]
+        for material_document_id in material_document_ids:
+            Report.result(
+                Tests.opened_original_new_child_material_documents,
+                atom_tools_utils.is_document_open(material_document_id) is True)
+
+        # 9. Verify the changes are saved in original_material_document.
+        Report.result(
+            Tests.saved_changes_to_original_material_document,
+            material_editor_utils.get_property(
+                original_material_document_id, base_color_property_name) == original_material_document_color)
+
+        # 10. Verify the changes are saved in new_material_document.
+        Report.result(
+            Tests.saved_changes_to_new_material_document,
+            material_editor_utils.get_property(
+                new_material_document_id, base_color_property_name) == new_material_document_color)
+
+        # 11. Verify the changes are saved in child_material_document.
+        Report.result(
+            Tests.saved_changes_to_child_material_document,
+            material_editor_utils.get_property(
+                child_material_document_id, base_color_property_name) == child_material_document_color)
+
+        # 12. Revert original_material_document color to starting_material_document_color and save it.
+        material_editor_utils.set_property(
+            original_material_document_id, base_color_property_name, starting_material_document_color)
+        Report.result(
+            Tests.reverted_original_material_document_color,
+            material_editor_utils.get_property(
+                original_material_document_id, base_color_property_name) == starting_material_document_color)
+        original_document_saved = atom_tools_utils.save_document(original_material_document_id)
+        Report.result(
+            Tests.resaved_original_material_document,
+            original_document_saved is True)
+
+        # 13. Close all currently opened documents.
+        closed_all_material_documents = atom_tools_utils.close_all_documents()
+        Report.result(
+            Tests.reclosed_original_new_child_material_documents,
+            closed_all_material_documents is True)
+        for material_document_id in material_document_ids:
```

Comment:
is there an open document count call we can make to confirm that document count is zero after close_all_documents()?

Reply:
not that I could see in the bindings